sotodlib/mapmaking/utilities.py

```diff
@@ -440,8 +443,68 @@ def rangemat_sum(rangemat):
         res[i] = np.sum(ra[:,1]-ra[:,0])
     return res
 
-def find_usable_detectors(obs, maxcut=0.1):
-    ncut  = rangemat_sum(obs.flags.glitch_flags)
+
+def parameter_in_path(obj: Any, rpath: str, sep: str = ".") -> bool:
```

Comment:
Given that `"multi.component.key" in some_axis_manager_instance` now works, do you need this function?

Reply:
I'm really sorry Matthew, I forgot to push the last sync that removes all these.